pset-iterators.js

```diff
@@ -184,6 +232,13 @@ console.log('Problem 7:')
 
 // Add your code below this line
 
+const shortTermPresidents = presidents.filter((objectPresident) => {
+  const lessThanFourYears = objectPresident.took_office.split(',')
+  return lessThanFourYears.length <= 4
+})
+
+console.log(shortTermPresidents)
+
 // Add your code above this line
 
```

Comment:
Sample solution to bonus problem 8 (using the reduce method to fill an empty object) can be found [here](https://github.com/jsd20200127/hw-03-iterators-solution/blob/c31d6d65d8f18a945cb6e59470aca10687d61aa3/pset-iterators.js#L244-L261).